android/src/main/java/com/mackentoch/beaconsandroid/BeaconsAndroidModule.java

```diff
@@ -501,6 +503,11 @@ Notification buildNotification(Context context, @NonNull ReadableMap notificatio
       .setPriority(priority)
       .setContentIntent(pendingIntent);
 
+    NotificationChannel channel = new NotificationChannel(
+      channelId, "Beacon Channel" , NotificationManager.IMPORTANCE_DEFAULT);
+    NotificationManager notificationManager = (NotificationManager)context.getSystemService(Context.NOTIFICATION_SERVICE);
+    notificationManager.createNotificationChannel(channel);
```

Comment:
The `createNotificationChannel` method is typically called once when the App is initialized. For example, it is called in the onCreate() method of the Application class.
I don't know what will happen if it is called continuously. It is better to ensure that it is called only once when initialization.

Reply:
@morimotosan 
You're right.
I fixed to create channel every channelId in [f3bd399](https://github.com/oviceinc/react-native-beacons-manager/pull/4/commits/f3bd399f60d329b7ba21985a85f5e5acbe6ce89e).